calendar/src/main/resources/application-prod.properties

```diff
@@ -1,3 +1,5 @@
 spring.datasource.url=jdbc:postgresql://calendar-database/calendar
 spring.security.oauth2.resourceserver.jwt.issuer-uri=http://keycloak:8443/realms/lolmatch
 spring.rabbitmq.addresses=rabbitmq
+spring.security.allowed.cors=http://localhost:4200
```

Comment:
Jeżeli to jest prod, to daj tu nazwe DNS po nazwie kontenera.

Reply:
A jaka jest nazwa kontenera?